property_tests/arbitraries/candid/simple_type_arbs/definition_arb.ts

```diff
@@ -22,6 +24,7 @@ export function SimpleCandidDefinitionArb(
                     candidTypeAnnotation,
                     candidTypeObject,
                     candidType,
+                    azleCandidTypeObject,
```

Comment:
Rename `azleCandidTypeObject` to `runtimeCandidTypeObject`, rename this everywhere appropriate